starcheck/utils.py

```diff
@@ -543,3 +544,20 @@ def proseco_probs(**kw):
     ]
 
     return p_acqs, float(-np.log10(acq_cat.calc_p_safe())), float(np.sum(p_acqs))
+
+
+def vehicle_filter_backstop(backstop_file, outfile):
```

Comment:
Ideally, we'd add a unit test for this function, but given the impact I think it can probably go with functional testing.